parcels/examples/tutorial_jit_vs_scipy.ipynb

```diff
@@ -74,7 +76,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As you can see above, **Scipy mode took more than 20 times as long** (17.2 seconds versus 0.79 seconds) as the JIT mode!!\n",
+    "As you can see above, **Scipy mode took almost 3 times as long** (7.3 seconds versus 2.6 seconds) as the JIT mode!!\n",
```

Comment:
We want to highlight that JIT is way faster than Scipy. Why don't take we more particles such that JIT time is not mostly initialisation.